test/format/salvage.c

```diff
@@ -69,24 +69,14 @@ corrupt(void)
      */
     testutil_check(__wt_snprintf(buf, sizeof(buf), "%s/%s", g.home, WT_NAME));
     if ((fd = open(buf, O_RDWR)) != -1) {
-#ifdef _WIN32
         testutil_check(__wt_snprintf(copycmd, sizeof(copycmd),
-          "copy %s\\%s %s\\slvg.copy\\%s.corrupted", g.home, WT_NAME, g.home, WT_NAME));
-#else
-        testutil_check(__wt_snprintf(copycmd, sizeof(copycmd), "cp %s/%s %s/slvg.copy/%s.corrupted",
-          g.home, WT_NAME, g.home, WT_NAME));
-#endif
+          "cp %s/%s %s/SALVAGE.copy/%s.corrupted", g.home, WT_NAME, g.home, WT_NAME));
```

Comment:
In `backup.c` you changed `BACKUP_COPY` to `BACKUP.COPY`. That's fine. This and backups should have the same `.copy` or `.COPY` naming I think. I don't care which one it is.

Reply:
Thanks. I was trying to (1) get the naming consistent instead of having random stuff scattered around the directory, and (2) give each chunk of code control over its own naming so it's easier for people not familiar with the overall code to modify their stuff. I've tried again, hopefully all fixed in 125de7feb.

It does kind of feel like we're shouting in the directory now, let me know if you have a better idea.